editor.planx.uk/src/@planx/components/AddressInput/AddressInput.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/testing-library";
+import React from "react";
+
+import Wrapper from "../fixtures/Wrapper";
+import Editor from "./Editor";
+import Public from "./Public";
+
+export default {
+  title: "PlanX Components/AddressInput",
+  component: Public,
+} as Meta;
```

Comment:
This file results in these stories, which is generally the pattern I've followed for all inputs: 
![Screenshot from 2023-07-05 13-45-02](https://github.com/theopensystemslab/planx-new/assets/5132349/ea1518d1-8fd4-420a-82bd-a7076cdc58ce)

`Filled Form` will automatically trigger validation errors. I think with another addon or two we could directly incorporate `expect` testing in here in the future too, but I haven't done that yet - and those types of scenarios are already typically tested via Jest in `Public.test.tsx` for each component. 

`With Editor` renders a `Wrapper` that allows you to fill out the editor modal, click "Update", then render the live "Preview" below. Storybook 7's new UI felt better suited to top-bottom layout here, but an easy one to come back to if side-by-side is preferred?